### ISSUE TITLE
Latest Pipeline Tool, 3.6.0.1625 cannot convert WAV file. Previous version could, 3.5.0.1678

### ISSUE THREAD

**Issue by fatcatz888** [MonoGame/MonoGame#5534]
MonoGame Pipeline Tool 3.5.0.1678 can convert to XNA the attached wav file. But the latest version,  cannot 3.6.0.1625. Latest version will throw an error.

[chip.zip](https://github.com/MonoGame/MonoGame/files/814126/chip.zip)

**Comment by cra0zy** [comment]
```
chip.wav: error: Importer 'WavImporter' had unexpected failure!
Microsoft.Xna.Framework.Content.Pipeline.InvalidContentException: Failed to open file chip. Ensure the file is a valid audio file and is not DRM protected. ---> System.InvalidOperationException: Probed audio format does not match RIFF
  at Microsoft.Xna.Framework.Content.Pipeline.Audio.AudioContent..ctor (System.String audioFileName, Microsoft.Xna.Framework.Content.Pipeline.Audio.AudioFileType audioFileType) [0x00167] in <775ee6b08f8441cd9946f7532ebffc51>:0 
   --- End of inner exception stack trace ---
  at Microsoft.Xna.Framework.Content.Pipeline.Audio.AudioContent..ctor (System.String audioFileName, Microsoft.Xna.Framework.Content.Pipeline.Audio.AudioFileType audioFileType) [0x001a0] in <775ee6b08f8441cd9946f7532ebffc51>:0 
  at Microsoft.Xna.Framework.Content.Pipeline.WavImporter.Import (System.String filename, Microsoft.Xna.Framework.Content.Pipeline.ContentImporterContext context) [0x00048] in <775ee6b08f8441cd9946f7532ebffc51>:0 
  at Microsoft.Xna.Framework.Content.Pipeline.ContentImporter`1[T].Microsoft.Xna.Framework.Content.Pipeline.IContentImporter.Import (System.String filename, Microsoft.Xna.Framework.Content.Pipeline.ContentImporterContext context) [0x00022] in <775ee6b08f8441cd9946f7532ebffc51>:0 
  at MonoGame.Framework.Content.Pipeline.Builder.PipelineManager.ProcessContent (MonoGame.Framework.Content.Pipeline.Builder.PipelineBuildEvent pipelineEvent) [0x00081] in <775ee6b08f8441cd9946f7532ebffc51>:0 
```

Important part:
```
Failed to open file chip. Ensure the file is a valid audio file and is not DRM protected.
```

**Comment by fatcatz888** [comment]
It is not DRM protected. Also, MonoGame Pipeline Tool 3.5.0.1678 can read the file without any problems. It's the new version that cannot read the file.

**Comment by KonajuGames** [comment]
I'm fixing this now.  Turns out your sample wav is 32-bit floating point mono.  I'm adding support for IEEE Float wav formats in the content pipeline and in the runtime for `SoundEffect.FromStream()`.  DirectX is done. Doing OpenAL now.

**Comment by KonajuGames** [comment]
OpenAL is a little more difficult.  Will finish it tomorrow.

**Comment by fatcatz888** [comment]
For the time being, I converted it to WAV (Microsoft) signed 16-bit PCM.

**Comment by monobeginner** [comment]
Is it fixed now ? Any stable release or workaround ?

**Comment by KonajuGames** [comment]
This should be all good now. Let us know if there are still any issues.

**Comment by monobeginner** [comment]
Thanks for your quick response. Actually I got MG 3.7.0.752 and WAV issue seems to be fixed. But I got another problem during build of Android project:
C:\Program Files (x86)\MSBuild\MonoGame\v3.0\MonoGame.Content.Builder.targets(73,5): error MSB3191: Unable to create directory "C:\Users\...". The given path's format is not supported.

It seems that this path is "duplicated" but I have no idea why ..
